docs/libssh2_channel_read_host_info.3

```diff
@@ -0,0 +1,36 @@
+.TH libssh2_channel_read_host_info 3 "23 Jan 2019" "libssh2 1.8.0" "libssh2 manual"
+.SH NAME
+libssh2_channel_read_host_info - read host information.
+.SH SYNOPSIS
+#include <libssh2.h>
+
+LIBSSH2_API int libssh2_channel_read_host_info(LIBSSH2_CHANNEL *channel,
+                                               unsigned char *shost,
+                                               size_t shost_buffer_size,
+                                               unsigned int *sport,
+                                               unsigned int *shost_len);
+
+.SH DESCRIPTION
+This function is used to pass shost, sport, and shost_len from library to user space application.
+underlying function \fIlibssh2_channel_read_ex(3)\fP.
+
+\fIchannel\fP - Active channel stream to read from.
+
+\fIshost\fP - Pointer to host to tell the application where the connection originated from.
+
+\fIshost_buffer_size\fP - Size of buffer allocated for shost.
+
+\fIsport\fP - Pointer to port to tell the application where the connection originated from.
```

Comment:
Elaborate. What is a port it originates from. Is this the TCP source port for the connection? If so, why not say so?